src/fsharp/FSharp.Core/control.fsi

```diff
@@ -268,6 +268,21 @@ namespace Microsoft.FSharp.Control
         /// <returns>A computation that returns an array of values from the sequence of input computations.</returns>
         static member Parallel : computations:seq<Async<'T>> -> Async<'T[]>
 
+        /// <summary>Creates an asynchronous computation that executes all given asynchronous computations in parallel, 
+        /// returning the result of the first succeeding computation (i.e. the first computation with a result that is 'Some x').</summary>
```

Comment:
I basically just copied the format from the corresponding Async.Parallel segment.